### ISSUE TITLE
expression def in figure 3

### ISSUE THREAD

**Issue by pnwamk** [jsiek/B629-denotational#4]
https://github.com/jsiek/B629-denotational/blob/c2cec309126f700140413fbff5dc6332066ef1a6/notes.tex#L381

TL;DR I think the definition for `expressions` in figure 3 is wrong... specifically the `...` implying we are extending something doesn't make sense to me.

Long: In figure 3 it seems we are defining `expressions` by extending  a previous definition and adding variables (that's how I read the `...`).

The previous definition for expressions (in section 1) included binary numerals whose metavariable is `b`, and in figure 3 we're also given the metavariable `b` to mean `conditions` (which are defined in figure 3).

So... I'm just trying to figure out if we still have binary numerals... but they (or conditions) need a different metavariable? or are we not extending the previous `expressions` we had?

**Comment by pnwamk** [comment]
I guess as well from our lecture today I had assumed we did not have binary numerals, but had `Nat`s and expressions over `Nat`s instead of binary numeral expressions, etc
